mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -466,6 +466,20 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 a
 		ARG 2 b
 		ARG 4 c
+	METHOD method_41303 catmullRom (FFFFF)F
+		COMMENT An implementation of Catmull-Rom Splines interpolation.
```

Comment:
```suggestion
		COMMENT Interpolates a point on a Catmull-Rom Spline. This spline has a property that if there are two
		COMMENT splines with arguments {@code p0, p1, p2, p3} and {@code p1, p2, p3, p4}, the resulting curve
		COMMENT will have a continuous first derivative at {@code p2}, where the two input curves connect. For
		COMMENT higher-dimensional curves, the interpolation on the curve is done component-wise: for
		COMMENT inputs {@code delta, (p0x, p0y), (p1x, p1y), (p2x, p2y), (p3x, p3y)}, the output is
		COMMENT {@code (catmullRom(delta, p0x, p1x, p2x, p3x), catmullRom(delta, p0y, p1y, p2y, p3y))}.
```

Reply:
Oh yeah! Your description is easier to understand.